build.cmd

```diff
@@ -106,7 +106,21 @@ if "%1" == "" goto ArgsDone
 if /i "%1" == "-?"    goto Usage
 if /i "%1" == "-h"    goto Usage
 if /i "%1" == "-help" goto Usage
+if /i "%1" == "--help" goto Usage
```

Comment:
I agree. Whenever I run build.sh --help in CoreFX, I always forget that this form is not supported.